sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/WebAppsImpl.java

```diff
@@ -142,8 +143,8 @@ private static boolean isWebApp(SiteInner inner) {
         if (inner.kind() == null) {
             ret = true;
         } else {
-            List<String> kinds = Arrays.asList(inner.kind().split(","));
-            if (kinds.contains("app") || kinds.contains("api")) {
+            List<String> kinds = Arrays.asList(inner.kind().split(Pattern.quote(",")));
+            if ((kinds.contains("app") || kinds.contains("api")) && !kinds.contains("kubernetes")) {
```

Comment:
Do we want to have enum/constants to represent kinds value?

Reply:
Currently they do not have a shared parent to put the constant, and it seems not worthwhile to make a new class for constant, as currently here is only 1 "kubernetes" duplicated in 2 places.

Reply:
ok, it's fine for now.